migtests/tests/analyze-schema/expected_issues.json

```diff
@@ -371,16 +371,6 @@
             "GH": "https://github.com/YugaByte/yugabyte-db/issues/1131",
             "MinimumVersionsFixedIn": null
         },
-        {
```

Comment:
how come this went away?

Reply:
as discussed - 
As I now added invalid count for this regex - 
```
else if spc := alterViewRegex.FindStringSubmatch(sqlInfo.stmt); spc != nil {
			summaryMap["VIEW"].invalidCount[sqlInfo.objName] = true
			reportCase(fpath, "ALTER VIEW not supported yet.",
				"https://github.com/YugaByte/yugabyte-db/issues/1131", "", "VIEW", spc[1], sqlInfo.formattedStmt, UNSUPPORTED_FEATURES, "")
		}
```
and hence this is not going in this condition - 
```
_, err := queryparser.Parse(sqlStmtInfo.stmt)
if err != nil { //if the Stmt is not already report by any of the regexes
	if !summaryMap[objType].invalidCount[sqlStmtInfo.objName] {
		reason := fmt.Sprintf("%s - '%s'", UNSUPPORTED_PG_SYNTAX, err.Error())
		reportCase(fpath, reason, "https://github.com/yugabyte/yb-voyager/issues/1625",
			"Fix the schema as per PG syntax", objType, sqlStmtInfo.objName, sqlStmtInfo.formattedStmt, UNSUPPORTED_FEATURES, "")
	}
```